zadani/soft/style.css

```diff
@@ -15,29 +16,311 @@ body {
     font-size: 16px;
     line-height: 1.6;
+    
 }
 
 
 /* HLAVICKA */
+.hlavicka {    
+    background-image: linear-gradient(rgba(0,0,0,0.3),rgba(0,0,0,0.3)), 
+                    url("../../images/island-hlavicka.jpg"); 
+    background-position: center;
+    background-size: cover;
+    min-height: 200px;
+    text-align: center;
+    padding: 5px;
+    margin: 0
+}
 
 .titulek {
-    font-family: 'Raleway', serif;
+    font-size: 50px;
+    font-family:'Raleway', serif;
+    color:white;
+    padding:20px;
+    margin:0px;
+}
+
+.perex {
+    font-size: 12px;
+    color:white;
+    padding:20px;
+    margin:0px;
+}
+
+.autor {
+    color: rgba(236, 233, 233, 0.836);
+    font-size: 12px;
+    font-style: italic;
+    padding:30px 20px 30px 20px;
+    position: relative;
 }
 
+.autor::after {
+    content: "";
+    width:18%;
+    border-top: 1px solid rgba(236, 233, 233, 0.863);
+    position: absolute;
+    left: 40%;
+    top:15%;
+}
 
 /* MENU */
 
+.menu {
+    font-size: 14px;
+    text-transform: uppercase;
+    background-color: white;
+    display:flex;
+    flex-direction: column;
+    text-align: center;
+    padding-right: 50px;
+    padding-left: 50px;
+    padding-right: 110px;
+}
+
+.menu_seznam li{
+    display: inline-block;
+    margin: 10px;
+}
 
+.menu_seznam a {
+    color: rgb(165, 162, 162);
+    text-decoration: none;
+}
 
 
 /* OBSAH */
+.obsah
+{
+    background-color: white;
+    box-shadow: 5px 12px 20px rgba(26, 22, 26, 0.2);
+    padding-bottom: 20px;
+}
 
+h2 {
+    padding:20px 0px;
+    margin: 0px; 
+    position: relative;
+}
 
+h2::after {
+    content: "";
+    width: 70px;
+    border-bottom: 4px solid rgb(95, 147, 190);
+    position:absolute;
+    bottom:0;
+    left: -30px;
+}
 
+.fotka_islandu  img{
+    width:100%;
+    margin: top 1px;
+    background-size: cover;
+}
+
+.strana_vlevo,
+.strana_vpravo {
+    padding: 15px 25px 15px 25px;
+    font-size: 15px;
+}
+
+.popisek,
+.popisek a{
+    color:rgb(165, 162, 162);
+    font-size: 10px;
+    text-align: center;
+    padding-top:5px;
+    margin:0px;
+}
 
 /* FOTOGALERIE */
 
 
+.galerie {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+.foto img  {
+    width:100%;
+    margin-top:  15px;
+    margin-bottom: 10px;
+    display:block;
+}
+
+.popisek_galerie {
+    font-size: 12px;
+    font-family: 'Raleway', serif;
+    font-weight: 700;
+    color:grey;
+    text-align: center;
+}
+.popisek-copy, 
+.popisek-copy a {
+    font-weight: 400;
+    color: rgb(165, 162, 162);
+    font-size: 10px;
+}
+
+.popisek-copy a {
+    text-decoration: underline;
+}
 
 
 /* PATICKA */
+.paticka,
+.paticka a {
+    color: gray;
+    font-size: 10px;
+    padding-bottom: 20px; 
+}
+
+.paticka
+{
+    background-color: #f5f5f54b;
+    font-size: 10px;
+    font-weight: 400;
+    padding: 20px;
+}
+
+
+/* tablet */
+@media only screen and (min-width:500px){
+    .autor::after {
+        width:10%;
+        left: 45%;  
```

Comment:
Tohle bychom si pak mohli ušetřit, když nastavíš šířku pevně a na vycentrování použiješ transform